### ISSUE TITLE
publicKey deprecation

### ISSUE THREAD

**Issue by thekeenant** [firelab-app/plaid_link#1]
The most recent updates to Plaid Link (Android SDK) seem to indicate the usage of a public key is being deprecated. 

Just a reminder to look into it.

**Comment by ayr-ton** [comment]
@thekeenant Are you working on this?

**Comment by thekeenant** [comment]
@ayr-ton Nope, I've taken a bit of a break from my personal project. Happy to get any help.

**Comment by ayr-ton** [comment]
@thekeenant Will check about that. I saw that all new accounts already have public_key disabled.
So for making it work with existing SDKs (Android and Web) I would need to update this behavior as well. Seeing how complicated would be changing this during iOS SDK implementation.

**Comment by ayushin** [comment]
yeah i am looking into this as well, perhaps merging this web implementation with another flutter plaid library>?

**Comment by ayushin** [comment]
My quick take on it:

https://github.com/firelab-app/plaid_link/pull/3
